src/Components/v2/__tests__/Carousel.test.tsx

```diff
@@ -13,8 +13,8 @@ describe("Carousel", () => {
       <MockBoot breakpoint="xs">
         <Carousel
           data={[{ name: "foo" }]}
-          render={props => {
-            return <div />
+          render={(_, index) => {
+            return <div key={index} />
```

Comment:
Since the render method simply returns a value, the key can be added inside of Carousel directly within the iterator. No need to pass additional data up through props.

Reply:
I didnt want to return any wrapper element surrounding the Carousel mapped objects because that wrapper takes specific stylings.

Reply:
https://github.com/artsy/reaction/blob/b186231b2b67d0c8b94b8151777b2b47c1ef5f02/src/Components/v2/CarouselV3.tsx#L316

can be wrapped with a 

`<Fragment key={index}>`

Reply:
Cool